src/docs/sphinx/developerGuide/Contributing/UsingDocker.rst

```diff
@@ -50,7 +50,7 @@ I like to do
 
 .. code-block:: console
 
-    docker run --cap-add=ALL -d --name ${REMOTE_DEV_IMG}-${VERSION} -p 64000:22 ${REMOTE_DEV_IMG}:${VERSION}
+    docker run --cap-add=SYS_PTRACE -d --name ${REMOTE_DEV_IMG}-${VERSION} -p 64000:22 ${REMOTE_DEV_IMG}:${VERSION}
 
```

Comment:
Should we take the opportunity to update to image for `ubuntu20.04` or `ubuntu22.04` as `ubuntu18.04` has been discarded ? Just in case s.o. copy paste steps without reading :p